dask_image.Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+
+FROM gpuci/miniconda-cuda:$CUDA_VER-devel-$LINUX_VER
+
+ARG CUDA_VER=11.2
```

Comment:
This should probably be 11.5 now?  I suppose it doesn't matter too much but I think gpuCI now runs in 11.5 images and relies on CEC for 11.X compatibility

Reply:
Yeah I wasn't sure when we should make this change; would you mind if I did this in a separate PR bumping `CUDA_VER` in the matrix?

https://github.com/rapidsai/dask-build-environment/blob/1b0ceeb3bd4b28dc9273ec71d320def17de7c068/ci/axis/dask.yaml#L6-L7

Reply:
Sounds good

Reply:
Opened up https://github.com/rapidsai/dask-build-environment/pull/24 to address this